nginx-php-mysql/docker-compose.yaml

```diff
@@ -0,0 +1,34 @@
+version: "3.8"
+services:
+  backend:
+    image: php:8.0.3-fpm-buster
+    secrets:
+      - db-password
+    depends_on: 
+      - db
+    volumes:
+      - ./backend:/var/www/html # this part seems repetitive (jb)
+  db:
+    image: mysql:8.0.19
+    command: '--default-authentication-plugin=mysql_native_password'
+    restart: always
+    secrets:
+      - db-password
+    volumes:
+      - db-data:/var/lib/mysql
+    environment:
+      - MYSQL_DATABASE=example
+      - MYSQL_ROOT_PASSWORD_FILE=/run/secrets/db-password
+  proxy:
+    build: proxy
+    ports:
+      - 80:80
+    depends_on: 
+      - backend
+    volumes:
+      - ./backend:/var/www/html # this part seems repetitive (jb)
```

Comment:
Here looks like you are sharing with `nginx` the same thing as php.
`nginx` should only serve directly the static content

Reply:
IMO, it needs to be that way so you can load a .PHP file to render a website. This is commonly how PHP is used in production environments. 

Do you suggest another way to achieve the same result?